npc/re/merchants/enchan_illusion_dungeons.txt

```diff
@@ -1704,3 +1705,35 @@ OnInit:
 	setunitdata .@npc_id, UNPC_HAIRCOLOR, 6;
 	end;
 }
+
+//============================================================
+//= Illusion Merchant
+//============================================================
+-	marketshop	market_resonance_stone	-1,100003:2000000:99999,100004:2000000:99999
+// prontera,88,113,5	script	Illusion Merchant#0829	MAGICIAN_H,{
+prontera,88,113,5	script	Illusion Merchant#0829	HIDDEN_NPC,{
+	mes "[Illusion Merchant]";
+	mes "I'm selling two random optional weapon grant scrolls that I made myself. You can choose the payment method you like.";
+	next;
+	switch( select( "Purchase with Zeny.", "Purchase with Illusion Stones" )) {
+		mes "[Illusion Merchant]";
```

Comment:
Its really strange that our script engine supports this, but please dont use this kind of syntax.